packages/services-store/src/actions/__tests__/profileAPI.test.ts

```diff
@@ -50,6 +56,115 @@ describe('Redux actions for `ProfileAPI`', () => {
     ]);
   });
 
+  it('dispatches the action of error in monitoring user authentication status', async () => {
+    ProfileAPI.getUserStatus.mockRejectedValue(new Error('error-getuserstatus'));
+    ProfileAPI.checkCloudCookie.mockRejectedValue(new Error('error-getuserstatus'));
+    ProfileAPI.checkCloudDocsAPI.mockRejectedValue(new Error('error-getuserstatus'));
+
+    const store = mockStore();
+    let caught;
+    try {
+      await store.dispatch(loadUserStatus(MASTHEAD_AUTH_METHOD.DEFAULT));
+    } catch (error) {
+      caught = error;
+    }
+    expect(caught?.message).toBe('error-getuserstatus');
+    expect(convertValue(store.getActions())).toEqual([
+      {
+        type: PROFILE_API_ACTION.SET_REQUEST_USER_STATUS_IN_PROGRESS,
+        request: 'PROMISE',
+      },
+      {
+        type: PROFILE_API_ACTION.SET_ERROR_REQUEST_USER_STATUS,
+        error: 'error-getuserstatus',
+      },
+    ]);
+
+    try {
+      await store.dispatch(loadUserStatus(MASTHEAD_AUTH_METHOD.COOKIE));
+    } catch (error) {
+      caught = error;
+    }
+    expect(caught?.message).toBe('error-getuserstatus');
+    expect(convertValue(store.getActions())).toEqual([
+      {
+        type: PROFILE_API_ACTION.SET_REQUEST_USER_STATUS_IN_PROGRESS,
+        request: 'PROMISE',
+      },
+      {
+        type: PROFILE_API_ACTION.SET_ERROR_REQUEST_USER_STATUS,
+        error: 'error-getuserstatus',
+      },
+    ]);
+
+    try {
+      await store.dispatch(loadUserStatus(MASTHEAD_AUTH_METHOD.DOCS_API));
+    } catch (error) {
+      caught = error;
+    }
+    expect(caught?.message).toBe('error-getuserstatus');
+    expect(convertValue(store.getActions())).toEqual([
+      {
+        type: PROFILE_API_ACTION.SET_REQUEST_USER_STATUS_IN_PROGRESS,
+        request: 'PROMISE',
+      },
+      {
+        type: PROFILE_API_ACTION.SET_ERROR_REQUEST_USER_STATUS,
+        error: 'error-getuserstatus',
+      },
+    ]);
+  });
+});
+
+describe('Redux actions for cloud implementation of `ProfileAPI`', () => {
+  it('dispatches the action to get user authentication status', async () => {
+    ProfileAPI.getUserStatus.mockResolvedValue({ user: CLOUD_UNAUTHENTICATED_STATUS });
+    const store = mockStore();
+    await store.dispatch(loadUserStatus(MASTHEAD_AUTH_METHOD.DEFAULT));
+    expect(convertValue(store.getActions())).toEqual([
+      {
+        type: PROFILE_API_ACTION.SET_REQUEST_USER_STATUS_IN_PROGRESS,
+        request: 'PROMISE',
+      },
+      {
+        type: PROFILE_API_ACTION.SET_USER_STATUS,
+        request: { user: CLOUD_UNAUTHENTICATED_STATUS },
+      },
+    ]);
+  });
```

Comment:
we can go ahead and remove the `it('dispatches the action to get user authentication status',` test here, it's redundant